.github/scripts/authors_in_cff.py

```diff
@@ -1,5 +1,10 @@
 """Check that the authors of a PR are included in ``CITATION.cff``."""
 
+# /// script
+# requires-python = ">=3.13"
+# dependencies = ["requests>=2.32"]
+# ///
+
```

Comment:
This follows the new standard for [inline script metadata](https://packaging.python.org/en/latest/specifications/inline-script-metadata/#inline-script-metadata).